vertx-web-client/src/main/java/io/vertx/ext/web/client/impl/HttpContext.java

```diff
@@ -196,6 +196,7 @@ private void sendRequest() {
     if (request.timeout > 0) {
       req.setTimeout(request.timeout);
     }
+    req.setHost(request.hostHeader);
```

Comment:
I think we should check wether the host is null or not before setting it

Reply:
I think we don't need to do it. The HttpClientRequestImpl that uses that value already checks whether the value is null or not, see here: https://github.com/eclipse/vert.x/blob/master/src/main/java/io/vertx/core/http/impl/HttpClientRequestImpl.java#L644